src/main/scala/format/pgn/Dumper.scala

```diff
@@ -43,18 +43,16 @@ object Dumper {
 
         s"${role.pgn}$disambiguation${if (captures) "x" else ""}${dest.key}"
     }) + {
-      if (next.check) {
-        if (next.checkMate) "#" else "+"
-      } else if (next.winner.isDefined) "#"
+      if (next.winner.isDefined) "#"
```

Comment:
I notice this code is duplicated, would it make sense to break this out into another function, or would we consider that over-engineered?

Maybe something like:

```
def apply(next: Situation): String = {
  // duplicated code goes here
}
```

Reply:
Seems unnecessary for such a simple duplication, extracting those two lines into a function probably just makes it harder to read.